spanner/session.go

```diff
@@ -149,6 +160,10 @@ func (sh *sessionHandle) getClient() *vkit.Client {
 	if sh.session == nil {
 		return nil
 	}
+	if sh.session.isMultiplexed {
```

Comment:
I think it would be safer/easier to read if we just check for `if sh.client != nil`. That also allows us to use the same field for regular sessions if that would ever be handy, and it is clear to anyone reading the code.